### ISSUE TITLE
Negative reputation

### ISSUE THREAD

**Issue by xsttx** [Bit-Nation/Lucy-draft#1]
https://github.com/Bit-Nation/Lucy-draft/blob/62799d0e0d9f65738a9c85c12615d7ffb8ca27a6/Lucy_DRAFT#L271

Lucy only assigns positive reputation. Failure to execute on a contract or solve a dispute simply translates into no reputation tokens, not negative ones.

As for the ability to down vote contracts, that's still a subject for debate. Imagine for instance if people started downvoting all contracts protecting assets of homeless people, as a coordinated attack, or something similar - it could be used as a weapon to target certain group. Rather it would be better with a function to upvote contracts if they fullfill their stated objective, and the ability to flaw contracts if there are technical issues with them.

**Comment by kentdahlgren** [comment]
You are correct. That's an oversight on my part. I'd sampled reputation code from git and it included a negative reputation value, which I intended to remove from the final release, but kept within the code so I could study how it functioned it its entirety. I will remove it soon.

**Comment by xsttx** [comment]
Got it. I'll keep the issue open though, because of the second point mentioned regarding negative reputation as an attack vector against groups when it comes to up voting and down voting smart contracts.
